src/platforms/common/profiling/index.mdx

```diff
@@ -249,28 +271,6 @@ The <PlatformIdentifier name="profiles_sample_rate" /> setting is _relative_ to
 
 </PlatformSection>
 
-<PlatformSection supported={["node"]}>
-
-```javascript
-import * as Sentry from "@sentry/node";
-import { ProfilingIntegration } from "@sentry/profiling-node";
-
-Sentry.init({
-  dsn: "___DSN___",
-  tracesSampleRate: 1,
-  profilesSampleRate: 1,
-  integrations: [new ProfilingIntegration()],
-});
-```
-
-<Note>
-
-The <PlatformIdentifier name="profilesSampleRate" /> setting is _relative_ to the <PlatformIdentifier name="tracesSampleRate" /> setting.
-
-</Note>
-
-</PlatformSection>
```

Comment:
Node profiling has it's own set of docs in `src/platforms/node/common/profiling` so this is unneeded.